src/lib/theme/colors.ts

```diff
@@ -157,6 +157,10 @@ const colors = {
   offWhite: '#F8FAFC',
   black: '#141414',
   offBlack: '#0F172A',
+  spaceGray: '#141414',
+  asteroidBeige: '#f8f8f7',
+  lightBlue: '#DFFBFF',
```

Comment:
Should we just find the nearest `blue` and `purple` for these two? `purple-100` and `blue-100` seem dangerously close.